test/Common/MoistThermodynamics/profiles.jl

```diff
@@ -0,0 +1,124 @@
+#### Tested moist thermodynamic profiles
+
+#=
+This file contains functions to compute all of the
+thermodynamic _states_ that MoistThermodynamics is
+tested with in runtests.jl
+=#
+
+
+"""
+    fixed_lapse_rate(
+        param_set::AbstractParameterSet,
+        z::FT,
+        T_surface::FT,
+        T_min::FT,
+        ) where {FT <: AbstractFloat}
+
+Fixed lapse rate hydrostatic reference state
+"""
+function fixed_lapse_rate(
+    param_set::AbstractParameterSet,
+    z::FT,
+    T_surface::FT,
+    T_min::FT,
+) where {FT <: AbstractFloat}
+    _grav::FT = grav(param_set)
+    _cp_d::FT = cp_d(param_set)
+    _R_d::FT = R_d(param_set)
+    _MSLP::FT = MSLP(param_set)
+    Γ = _grav / _cp_d
+    z_tropopause = (T_surface - T_min) / Γ
+    H_min = _R_d * T_min / _grav
+    T = max(T_surface - Γ * z, T_min)
+    p = _MSLP * (T / T_surface)^(_grav / (_R_d * Γ))
+    T == T_min && (p = p * exp(-(z - z_tropopause) / H_min))
```

Comment:
```suggestion
    T_virt == T_min && (p = p * exp(-(z - z_tropopause) / H_min))
```
(and analogously throughout, all of these are virtual temperatures)